test/parallel/test-repl.js

```diff
@@ -171,30 +171,31 @@ function error_test() {
     // invalid input to JSON.parse error is special case of syntax error,
     // should throw
     { client: client_unix, send: 'JSON.parse(\'{invalid: \\\'json\\\'}\');',
-      expect: /\bSyntaxError: Unexpected token i/ },
+      expect: /^SyntaxError: Unexpected token i/ },
     // end of input to JSON.parse error is special case of syntax error,
     // should throw
     { client: client_unix, send: 'JSON.parse(\'066\');',
-      expect: /\bSyntaxError: Unexpected number/ },
+      expect: /^SyntaxError: Unexpected number/ },
     // should throw
     { client: client_unix, send: 'JSON.parse(\'{\');',
-      expect: /\bSyntaxError: Unexpected end of JSON input/ },
+      expect: /^SyntaxError: Unexpected end of JSON input/ },
     // invalid RegExps are a special case of syntax error,
     // should throw
     { client: client_unix, send: '/(/;',
-      expect: /\bSyntaxError: Invalid regular expression:/ },
+      expect: /^SyntaxError: Invalid regular expression:/ },
     // invalid RegExp modifiers are a special case of syntax error,
     // should throw (GH-4012)
     { client: client_unix, send: 'new RegExp("foo", "wrong modifier");',
-      expect: /\bSyntaxError: Invalid flags supplied to RegExp constructor/ },
+      expect: /^SyntaxError: Invalid flags supplied to RegExp constructor/ },
     // strict mode syntax errors should be caught (GH-5178)
     { client: client_unix,
       send: '(function() { "use strict"; return 0755; })()',
       expect: /\bSyntaxError: Octal literals are not allowed in strict mode/ },
     {
       client: client_unix,
       send: '(function(a, a, b) { "use strict"; return a + b + c; })()',
-      expect: /\bSyntaxError: Duplicate parameter name not allowed in this context/
+      expect:
+        /\bSyntaxError: Duplicate parameter name not allowed in this context/
```

Comment:
Some error messages quote erroneous code before the error message itself, so for these messages, we can't use `^` anchor (or we should add the `/m` flag beside).